### PR TITLE
Allow up to 255 bots

### PR DESCRIPTION
**Pull request by ItsiAdam**

### CODE REVIEW

File: src/ipc.cpp [hunk 1]
```diff
@@ -133,7 +133,7 @@ CatCommand debug_get_ingame_ipc("ipc_debug_dump_server", "Show other bots on ser
     int count        = 0;
     unsigned highest = 0;
     std::vector<unsigned> botlist{};
-    for (unsigned i = 1; i < cat_ipc::max_peers; i++)
+    for (unsigned i = 0; i < cat_ipc::max_peers; i++)
```
Comment (TotallyNotElite) [inline src/ipc.cpp:136]: Don't include the world entity wtf

Comment (ItsiAdam) [reply src/ipc.cpp:136]: idk if you're joking but this is looping over the ipc peers, not entities

**Comment by ItsiAdam** [comment]
requires nullworks/simple-ipc#5  nullworks/cathook-ipc-web-panel#11 and  nullworks/cathook-ipc-server#9

**Comment by BenCat07** [comment]
I've looked at this before but we would have to merge all of them at once and update simple-ipc submodule paths too

**Comment by BenCat07** [comment]
Just updated the submodule, so i will reapprove for @TotallyNotElite's time's sake

**Pull request merged**